sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,649 @@
+            whether to force last values in one training validation to be NaN.
+            Helps make more robust models. defaults to None, which introduces NaN in
+            last rows of validations if any NaN in tail of training data. Will not
+            introduce NaN to all series if subset is used. if True, will also randomly
+            change 20% of all rows to NaN in the validations
+        preclean (dict):
+            The parameters for data pre-cleaning.
+            if not None, a dictionary of Transformer params to be applied to input data
+            {“fillna”: “median”, “transformations”: {}, “transformation_params”: {}}
+            This will change data used in model inputs for fit and predict, and for
+            accuracy evaluation in cross validation!
+        model_interrupt (bool):
+            Whether the model can be interrupted.
+            If False, KeyboardInterrupts quit entire program.
+            if True, KeyboardInterrupts attempt to only quit current model.
+            if True, recommend use in conjunction with verbose > 0 and result_file in
+            the event of accidental complete termination. if “end_generation”, as True
+            and also ends entire generation of run. Note skipped models will not be
+            tried again.
+        generation_timeout (int):
+            The timeout for each generation. if not None, this is the number of minutes
+            from start at which the generational search ends, then proceeding to
+            validation This is only checked after the end of each generation, so
+            only offers an ‘approximate’ timeout for searching. It is an overall
+            cap for total generation search time, not per generation.
+        current_model_file (str):
+            The file containing the current model.
+            file path to write to disk of current model params
+            (for debugging if computer crashes). .json is appended
+        verbose (int):
+            The verbosity level.
+            setting to 0 or lower should reduce most output.
+            Higher numbers give more output.
+        n_jobs (int):
+            The number of jobs to run in parallel.
+             Number of cores available to pass to parallel processing.
+             A joblib context manager can be used instead (pass None in this case).
+             Also ‘auto’.
+    """
+
+    _tags = {
+        "scitype:y": "univariate",
+        "authors": [
+            "MBristle",
+        ],
+        # "maintainers": ["MBristle"],
+        "y_inner_mtype": "pd.DataFrame",
+        "X_inner_mtype": "pd.DataFrame",
+        "ignores-exogeneous-X": True,  # TODO: add capability
+        "capability:insample": False,
+        "capability:pred_int:insample": False,
+        "capability:pred_int": False,  # TODO: add capability
+        "requires-fh-in-fit": True,
+        "python_dependencies": ["autots", "pandas", "sklearn", "statsmodels", "scipy"],
```

Comment:
```suggestion
        "python_dependencies": ["autots"],
```

Reason for this suggestion:

1. pandas, scipy are already sktime core dependencies. You don't need to add separately unless you need a specific version or higher.
2. same for scikit-learn, plus sklearn is not the PyPI name.
3. statsmodels - this I'm not confident of - where do you need this? Unless you are using it yourself as opposed this being a dependency of `autots`, it need not be specified.